src/jsrs_parser.cc

```diff
@@ -456,8 +463,10 @@ MaybeLocal<Value> ParseString(Isolate*    isolate,
 
     if (begin[i] == '\\') {
       if (IsLineTerminatorSequence(begin + i + 1, &in_offset)) {
+        use_fallback(i - 1);
```

Comment:
@aqrln 
Yeah, after looking at it one more time it looks okay, I changed it like you proposed.